client/src/app/layout.tsx

```diff
@@ -3,6 +3,7 @@ import { ReactNode } from "react";
 import type { Metadata } from "next";
 import { Oswald, Source_Sans_3 } from "next/font/google";
 import "./globals.css";
+import { Footer } from "@/components/layout/footer/footer";
```

Comment:
As we did with the header, when the file contains the same name as the folder, let's rename to `index.tsx` so the import becomes simpler.